source/code/plugin/lib/application_insights/channel/asynchronous_sender.rb

```diff
@@ -20,15 +20,16 @@ class AsynchronousSender < SenderBase
       SERVICE_ENDPOINT_URI = 'https://dc.services.visualstudio.com/v2/track'
       # Initializes a new instance of the class.
       # @param [String] service_endpoint_uri the address of the service to send
+      # @param [Hash] proxy server configuration to send (optional)
       #   telemetry data to.
-      def initialize(service_endpoint_uri = SERVICE_ENDPOINT_URI)
+      def initialize(service_endpoint_uri = SERVICE_ENDPOINT_URI, proxy = {})
```

Comment:
You can set the endpoint here from where its defined in AI SDK.

Reply:
Done.